pom.xml

```diff
@@ -58,5 +58,6 @@
   </properties>
   <modules>
     <module>syntax-markdown-markdown12</module>
+    <module>syntax-markdown-flavor</module>
```

Comment:
Is that structure something you discussed with @vmassol? I'm a bit surprised about this module: I'd have expected more to have all flavours at the same level as the standard one.

Reply:
No, I thought it would be good to include all other flavors in `syntax-markdown-flavor` parent.

Reply:
Should I make them on the top level as `markdown/1.2`?

Reply:
> Should I make them on the top level as markdown/1.2?

This should be discussed on the chat IMO, I'm not familiar with creating an extension for syntaxes so not sure. I think I'd personally add them at the top level, since they are all different flavors: the default one is just a specific type of flavor we chose to be the default. Now maybe I'm wrong here.

Reply:
I actually did ask on the chat [here](https://matrix.to/#/!ikPtGZaGWtyblizzlR:matrix.xwiki.com/$1596455394264005RSFuD:matrix.org?via=matrix.org&via=matrix.xwiki.com) (see point 1) but did not get any feedback :/

Reply:
yeah apparently your message has not been seen... Just asked again, and apparently @tmortagne confirms my feeling. Would be better to have all flavours at top level, along with the original syntax.